python/hsfs/feature_view.py

```diff
@@ -1502,9 +1488,6 @@ def create_train_test_split(
         statistics_config: Optional[Union[StatisticsConfig, bool, dict]] = None,
         write_options: Optional[Dict[Any, Any]] = None,
         spine: Optional[SplineDataFrameTypes] = None,
-        primary_key: bool = False,
```

Comment:
I think removing the arguments would be a breaking change, so I am not really sure if we can do it. Would it be better to change the default value of the arguments from `False` to `True`.